src/plugins/workspace/common/constants.ts

```diff
@@ -6,7 +6,13 @@
 export const WORKSPACE_FATAL_ERROR_APP_ID = 'workspace_fatal_error';
 export const WORKSPACE_CREATE_APP_ID = 'workspace_create';
 export const WORKSPACE_LIST_APP_ID = 'workspace_list';
+export const WORKSPACE_UPDATE_APP_ID = 'workspace_update';
 export const WORKSPACE_OVERVIEW_APP_ID = 'workspace_overview';
+/**
+ * Since every workspace always have overview and update page, these features will be selected by default
+ * and can't be changed in the workspace form feature selector
```

Comment:
instead of display them in disabled state, shall we just hide them?

Reply:
Does that means hide them on the workspace form feature selector?

Reply:
yes, I think this makes more sense to me, these are workspace fundamentals that each workspace should have.

Reply:
Updated.